### PR TITLE
Add operating system module

### PR DESCRIPTION
**Pull request by lukas-bednar**
Trying to solve #27 issue

### CODE REVIEW

File: rrmngmnt/operatingsystem.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+    @property
+    def release_str(self):
+        if not self._release_str:
+            self._release_str = self.get_release_str()
+        return str(self._release_str)
+
+    def get_release_info(self):
+        """
+        /etc/os-release bacame to be standard on systemd based operating
```
Comment (lukas-bednar) [inline rrmngmnt/operatingsystem.py]: s/bacame/became/

**Comment by lukas-bednar** [comment]
@vkondula @petr-balogh @tareqalayan @kobihk  could you please take a look? I welcome any suggestions / improvements.
it looks like it is not very deterministic to recognize current operating system ...

**Comment by petr-balogh** [comment]
Looks good for me. Maybe add to docstring more info that some methods are supported only for systems with systemd.

**Comment by lukas-bednar** [comment]
@petr-balogh I appreciate your input! I covered your comments.

**Pull request merged**